src/components/content/policies/EditPolicy.tsx

```diff
@@ -0,0 +1,280 @@
+        return false;
+    };
+
+    const onRemovePolicyContentFile = () => {
+        files.current.pop();
+        policyContent.current = '';
+        setRegoSyntaxValidationStatus('notStarted');
+        updatePoliciesManagementServiceRequest.reset();
+    };
+
+    const setValidator = (_: unknown, uploadFiles: UploadFile[]) => {
+        if (uploadFiles.length === 0) {
+            return Promise.reject(new Error('please upload policy file!'));
+        }
+        if (uploadFiles.length > 0) {
+            return Promise.resolve();
+        }
+    };
+
+    return (
+        <>
+            {createPolicyRequest !== undefined && createPolicyRequest.policy.length > 0 ? (
+                <PolicyCreateResultStatus
+                    isError={createPoliciesManagementServiceRequest.isError}
+                    isSuccess={createPoliciesManagementServiceRequest.isSuccess}
+                    error={createPoliciesManagementServiceRequest.error}
+                    currentPolicyService={createPoliciesManagementServiceRequest.data}
+                />
+            ) : null}
+            {updatePolicyRequest !== undefined && updatePolicyRequest.id.length > 0 ? (
+                <PolicyUpdateResultStatus
+                    isError={updatePoliciesManagementServiceRequest.isError}
+                    isSuccess={updatePoliciesManagementServiceRequest.isSuccess}
+                    error={updatePoliciesManagementServiceRequest.error}
+                    currentPolicyService={updatePoliciesManagementServiceRequest.data}
+                />
+            ) : null}
+            <Form
+                form={form}
+                name='basic'
+                labelCol={{ span: 8 }}
+                wrapperCol={{ span: 16 }}
+                className={'edit-form-class'}
+                initialValues={{
+                    remember: true,
+                    csp:
+                        currentPolicyService !== undefined && currentPolicyService.id.length > 0
+                            ? currentPolicyService.csp
+                            : undefined,
+                    enabled:
+                        currentPolicyService !== undefined && currentPolicyService.id.length > 0
+                            ? currentPolicyService.enabled
+                            : false,
+                    policy: undefined,
+                }}
+                onFinish={onFinish}
+                autoComplete='off'
+            >
+                <Form.Item label='Csp' name='csp' rules={[{ required: true, message: 'Please Select Csp!' }]}>
+                    <Select onSelect={handleCspSelect} size={'large'}>
+                        {Object.values(CredentialVariables.csp).map((csp: CredentialVariables.csp) => {
+                            return (
+                                <Select.Option key={csp} value={csp} className={'credential-select-option-csp'}>
+                                    <Image
```

Comment:
the images of CSP is not correctly placed in the drop down. They are not exactly at the center.